src/main/java/seedu/address/logic/commands/EventCommand.java

```diff
@@ -17,6 +18,7 @@
 import seedu.address.logic.commands.exceptions.CommandException;
 import seedu.address.model.Model;
 import seedu.address.model.event.Event;
+import seedu.address.model.person.Address;
```

Comment:
Can probably change the class name from `Address` to `Location` as well

Reply:
The Address class was the one that came with AB3 so I think it would be better to leave it? Since it's used by Person as well (for their own address).

Reply:
It's not used in `Person` anymore.